rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -323,6 +323,38 @@ public void getLockingCap() {
         verify(provider, times(1)).setLockingCap(constants.getInitialLockingCap());
     }
 
+    @Test
+    public void getActivePowpegRedeemScript_before_RSKIP293_activation() {
+        ActivationConfig.ForBlock activations = mock(ActivationConfig.ForBlock.class);
+        when(activations.isActive(ConsensusRule.RSKIP293)).thenReturn(false);
+
+        BridgeSupport bridgeSupport = bridgeSupportBuilder
+                .withBridgeConstants(bridgeConstants)
+                .withActivations(activations)
+                .build();
+
+        assertEquals(0, bridgeSupport.getActivePowpegRedeemScript().getProgram().length);
+    }
+
+    @Test
+    public void getActivePowpegRedeemScript_after_RSKIP293_activation() {
+        ActivationConfig.ForBlock activations = mock(ActivationConfig.ForBlock.class);
+        when(activations.isActive(ConsensusRule.RSKIP293)).thenReturn(true);
+
+        BridgeSupport bridgeSupport = bridgeSupportBuilder
+                .withBridgeConstants(bridgeConstants)
+                .withActivations(activations)
+                .build();
+
+
+        assertTrue(
+                Arrays.equals(
+                        bridgeConstants.getGenesisFederation().getRedeemScript().getProgram(),
+                        bridgeSupport.getActivePowpegRedeemScript().getProgram()
+                )
+        );
```

Comment:
```suggestion
        assertEquals(                     bridgeConstants.getGenesisFederation().getRedeemScript(),
                        bridgeSupport.getActivePowpegRedeemScript());
```
You can simply use the Script class equals

Reply:
It looks simpler. I'll do.